packages/notifi-dapp-example/src/app/notifi/dashboard/layout.tsx

```diff
@@ -36,5 +37,32 @@ export default function NotifiDashboardLayout({
     return null;
   }
 
+  const { frontendClient } = useNotifiClientContext();
+
+  const { isClientInitialized, isClientAuthenticated } = useMemo(() => {
+    return {
```

Comment:
Writting this way is kind of legacy pattern, we might wan to use `frontendClientStatus`